packages/manager/modules/pci/src/components/project/instance/instance.class.js

```diff
@@ -144,6 +144,10 @@ export default class Instance {
     return get(this, 'image.type') === 'windows' ? `rdekstop ${ip}` : `ssh ${user}@${ip}`;
   }
 
+  isApplicationImage() {
```

Comment:
We already have such logic here https://github.com/ovh-ux/manager/blob/698447521eb727e3025c7ce82e6730dda43c22fe/packages/manager/modules/pci/src/components/project/images-list/images.class.js#L27
I don't know if it is possible to avoid duplication in that case